web/soubory/blackarrow/program/script.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * @typedef { import("../_komponenty/multiselect/MultiselectButtons") }
+ */
+
+
+
+
+
+/**
+ * @typedef {Object} Aktivita
+ * @property {string} nazev
+ * @property {string} kratkyPopis
+ * @property {string} popis
+ * @property {string} obrazek
+ * @property {string[]} vypraveci
+ * @property {string[]} stitky
+ * @property {string} cena
+ * @property {string} cas
+ * @property {string} obsazenost
+ */
+
+
+// http://localhost/gamecon/web/api/program-roky
+
+
+/**
+ * @param {"muj" | "vedu" | number} view
+ * @returns {Promise<Aktivita[]>}
+ */
+const fetchAktivity = async (view) => {
+  const url = (typeof view === "number")
+    ? `http://localhost/gamecon/web/api/program?view=rok&rok=${view}`
```

Comment:
Nezapomeň tu URL změnit na dynamickou podle aktuální domény.

Můžeš to vytáhnout třeba z `<base href="...">`